UnityMirror/UnityServer/Assets/ScriptTemplates.meta

```diff
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
```

Comment:
This file was not needed to be commited in the first place or is it redundant now?

Reply:
It's a .meta file, which Unity had created for a folder and I didn't remove in a previous PR